editor/plugins/camera_editor_plugin.cpp

```diff
@@ -33,30 +33,18 @@
 #include "spatial_editor_plugin.h"
 
 void CameraEditor::_notification(int p_what) {
-
-	switch (p_what) {
-
-		/*		case NOTIFICATION_PROCESS: {
-
-			if (preview->is_pressed() && node)
-				node->call("make_current");
-
-		} break;*/
-	}
 }
+
 void CameraEditor::_node_removed(Node *p_node) {
 
 	if (p_node == node) {
 		node = NULL;
-		SpatialEditor::get_singleton()->set_custom_camera(NULL);
 		hide();
 	}
 }
 
 void CameraEditor::_pressed() {
 
```

Comment:
`clang-format` complains about the empty line here: https://travis-ci.org/godotengine/godot/jobs/423221752

Reply:
That said, if the `_pressed` callback is empty now, does this class do *anything*?